Comparing source compatibility of /home/someone/git/com.github/io7m/smfj/com.io7m.smfj.core/target/com.io7m.smfj.core-0.11.0.jar against /home/someone/.m2/repository/com/io7m/smfj/com.io7m.smfj.core/0.11.0-SNAPSHOT/com.io7m.smfj.core-0.11.0-SNAPSHOT.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
***  MODIFIED INTERFACE: PUBLIC ABSTRACT com.io7m.smfj.core.SMFCoordinateSystemType  (not serializable)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) com.io7m.jcoords.core.conversion.CAxisSystem axes()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) com.io7m.smfj.core.SMFFaceWindingOrder windingOrder()
**** MODIFIED INTERFACE: PUBLIC ABSTRACT com.io7m.smfj.core.SMFFormatVersionType  (not serializable)
	+++* NEW METHOD: PUBLIC(+) java.lang.String toHumanString()
***! MODIFIED CLASS: PUBLIC FINAL com.io7m.smfj.core.SMFHeader  (not serializable)
	===! UNCHANGED INTERFACE: com.io7m.smfj.core.SMFHeaderType
	---  REMOVED METHOD: PUBLIC(-) long metaCount()
	---! REMOVED METHOD: PUBLIC(-) STATIC(-) com.io7m.smfj.core.SMFHeader of(com.io7m.smfj.core.SMFTriangles, long, com.io7m.smfj.core.SMFSchemaIdentifier, com.io7m.smfj.core.SMFCoordinateSystem, javaslang.collection.List, long)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFHeader of(com.io7m.smfj.core.SMFTriangles, long, java.util.Optional, com.io7m.smfj.core.SMFCoordinateSystem, javaslang.collection.List)
	***! MODIFIED METHOD: PUBLIC java.util.Optional (<-com.io7m.smfj.core.SMFSchemaIdentifier) schemaIdentifier()
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFHeader withMetaCount(long)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFHeader withSchemaIdentifier(java.util.Optional)
***! MODIFIED CLASS: PUBLIC STATIC FINAL com.io7m.smfj.core.SMFHeader$Builder  (not serializable)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFHeader$Builder setMetaCount(long)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFHeader$Builder setSchemaIdentifier(java.util.Optional)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT com.io7m.smfj.core.SMFHeaderType  (not serializable)
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) com.io7m.smfj.core.SMFCoordinateSystem coordinateSystem()
	---! REMOVED METHOD: PUBLIC(-) long metaCount()
	***! MODIFIED METHOD: PUBLIC ABSTRACT java.util.Optional (<-com.io7m.smfj.core.SMFSchemaIdentifier) schemaIdentifier()
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) com.io7m.smfj.core.SMFTriangles triangles()
***! MODIFIED CLASS: PUBLIC FINAL com.io7m.smfj.core.SMFSchemaIdentifier  (not serializable)
	===! UNCHANGED INTERFACE: com.io7m.smfj.core.SMFSchemaIdentifierType
	+++  NEW METHOD: PUBLIC(+) com.io7m.smfj.core.SMFSchemaName name()
	---! REMOVED METHOD: PUBLIC(-) STATIC(-) com.io7m.smfj.core.SMFSchemaIdentifier of(int, int, int, int)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFSchemaIdentifier of(com.io7m.smfj.core.SMFSchemaName, int, int)
	---  REMOVED METHOD: PUBLIC(-) int schemaID()
	---  REMOVED METHOD: PUBLIC(-) int schemaMajorVersion()
	---  REMOVED METHOD: PUBLIC(-) int schemaMinorVersion()
	---  REMOVED METHOD: PUBLIC(-) int vendorID()
	+++  NEW METHOD: PUBLIC(+) int versionMajor()
	+++  NEW METHOD: PUBLIC(+) int versionMinor()
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier withName(com.io7m.smfj.core.SMFSchemaName)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier withSchemaID(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier withSchemaMajorVersion(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier withSchemaMinorVersion(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier withVendorID(int)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier withVersionMajor(int)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier withVersionMinor(int)
***! MODIFIED CLASS: PUBLIC STATIC FINAL com.io7m.smfj.core.SMFSchemaIdentifier$Builder  (not serializable)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setName(com.io7m.smfj.core.SMFSchemaName)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setSchemaID(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setSchemaMajorVersion(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setSchemaMinorVersion(int)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setVendorID(int)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setVersionMajor(int)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaIdentifier$Builder setVersionMinor(int)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT com.io7m.smfj.core.SMFSchemaIdentifierType  (not serializable)
	+++* NEW METHOD: PUBLIC(+) ABSTRACT(+) com.io7m.smfj.core.SMFSchemaName name()
	---! REMOVED METHOD: PUBLIC(-) int schemaID()
	---! REMOVED METHOD: PUBLIC(-) int schemaMajorVersion()
	---! REMOVED METHOD: PUBLIC(-) int schemaMinorVersion()
	---! REMOVED METHOD: PUBLIC(-) int vendorID()
	+++* NEW METHOD: PUBLIC(+) int versionMajor()
	+++* NEW METHOD: PUBLIC(+) int versionMinor()
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaName  (not serializable)
	+++  NEW INTERFACE: com.io7m.smfj.core.SMFSchemaNameType
	+++  NEW INTERFACE: java.lang.Comparable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFSchemaName$Builder builder()
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFSchemaName copyOf(com.io7m.smfj.core.SMFSchemaNameType)
	+++  NEW METHOD: PUBLIC(+) boolean equals(java.lang.Object)
	+++  NEW METHOD: PUBLIC(+) int hashCode()
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFSchemaName of(java.lang.String)
	+++  NEW METHOD: PUBLIC(+) java.lang.String toString()
	+++  NEW METHOD: PUBLIC(+) java.lang.String value()
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaName withValue(java.lang.String)
+++  NEW CLASS: PUBLIC(+) STATIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaName$Builder  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) com.io7m.smfj.core.SMFSchemaName build()
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaName$Builder from(com.io7m.smfj.core.SMFSchemaNameType)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaName$Builder setValue(java.lang.String)
+++  NEW CLASS: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFSchemaNames  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.regex.Pattern PATTERN
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int MAXIMUM_CHARACTERS
	+++  NEW METHOD: PUBLIC(+) STATIC(+) boolean isValid(java.lang.CharSequence)
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) com.io7m.smfj.core.SMFSchemaNameType  (not serializable)
	+++  NEW INTERFACE: java.lang.Comparable
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) void checkPreconditions()
	+++  NEW METHOD: PUBLIC(+) int compareTo(com.io7m.smfj.core.SMFSchemaNameType)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String value()
***! MODIFIED CLASS: PUBLIC FINAL com.io7m.smfj.core.SMFTriangles  (not serializable)
	===! UNCHANGED INTERFACE: com.io7m.smfj.core.SMFTrianglesType
	---! REMOVED METHOD: PUBLIC(-) STATIC(-) com.io7m.smfj.core.SMFTriangles of(long, long)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) com.io7m.smfj.core.SMFTriangles of(long, int)
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleIndexSizeBits()
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleIndexSizeOctets()
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleSizeOctets()
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFTriangles withTriangleIndexSizeBits(long)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFTriangles withTriangleIndexSizeBits(int)
***! MODIFIED CLASS: PUBLIC STATIC FINAL com.io7m.smfj.core.SMFTriangles$Builder  (not serializable)
	---! REMOVED METHOD: PUBLIC(-) FINAL(-) com.io7m.smfj.core.SMFTriangles$Builder setTriangleIndexSizeBits(long)
	+++  NEW METHOD: PUBLIC(+) FINAL(+) com.io7m.smfj.core.SMFTriangles$Builder setTriangleIndexSizeBits(int)
***! MODIFIED INTERFACE: PUBLIC ABSTRACT com.io7m.smfj.core.SMFTrianglesType  (not serializable)
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleIndexSizeBits()
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleIndexSizeOctets()
	***! MODIFIED METHOD: PUBLIC int (<-long) triangleSizeOctets()
+++  NEW INTERFACE: PUBLIC(+) ABSTRACT(+) com.io7m.smfj.core.SMFWarningType  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.util.Optional exception()
	+++  NEW METHOD: PUBLIC(+) java.lang.String fullMessage()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.String message()
